src/content/blog/maintainer-job-description.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "We're Recruiting DDEV Open Source Project Maintainers"
```

Comment:
Could say it more simply and use the right quote. (Smartypants isn’t applied to titles or meta details.)

```suggestion
title: "We’re Recruiting Project Maintainers"
```

We’re on ddev.com with the DDEV name immediately above this, with `ddev` in the URL. And it’s an open source project so that part’s probably redundant as well.